src/cli/utils.ts

```diff
@@ -139,4 +140,26 @@ export const createFiles = (filesToGenerate: FileToGenerate[]): void => {
     fs.writeFileSync(fileToGenerate.path, fileToGenerate.source);
   });
 };
+
+/**
+ * Translates `path` with `routeTranslations` if `lang` exists in
+ * `routeTranslations`, else returns `[base, path].join("/")`.
+ * @param base defaults to `""`.
+ */
+export const createTranslatedPath = (
+  routeTranslations: AstroI18nextConfig["routes"],
+  path: string,
+  lang?: string,
```

Comment:
For consistency, this should be named language. Please change it everywhere, the project uses explicit naming.
```suggestion
  language?: string,
```